schemas/build_params.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Parameters for a koji containerBuild task.",
+
+  "type": "array",
+  "items": [
+    {
+      "type": "string",
+      "description": "Source URI."
+    },
+    {
+      "type": "string",
+      "description": "Build target."
+    },
+    {
+      "type": ["object", "null"],
+      "properties": {
+        "scratch": {
+          "type": "boolean",
+          "description": "Perform a scratch build?"
+        },
+        "isolated": {
+          "type": "boolean",
+          "description": "Perform an isolated build?"
+        },
+        "yum_repourls": {
+          "type": ["array", "null"],
+          "items": {
+            "type": "string"
+          },
+          "description": "URLs of yum repo files."
+        },
+        "git_branch": {
+          "type": ["string", "null"],
+          "description": "Git branch to build from."
+        },
+        "push_url": {
+          "type": ["string", "null"]
+        },
+        "koji_parent_build": {
+          "type": ["string", "null"],
+          "description": "Overwrite parent image with image from koji build."
+        },
+        "release": {
+          "type": ["string", "null"],
+          "description": "Set release value."
+        },
+        "flatpak": {
+          "type": "boolean",
+          "description": "Build a flatpak instead of a container?"
+        },
+        "compose_ids": {
+          "type": ["array", "null"],
+          "items": {
+            "type": "integer"
+          },
+          "description": "ODCS composes used."
+        },
+        "signing_intent": {
+          "type": ["string", "null"],
+          "description": "Signing intent of the ODCS composes."
+        }
```

Comment:
Don't we need here to allow extra properties?

Reply:
I think extra properties are allowed by default (there is a test for it, and it passes)